.github/workflows/render_nanomaps.yml

```diff
@@ -0,0 +1,35 @@
+# GitHub action to autorender nanomaps outside the game
+# This kills off the awful verb we have that takes a full 50 seconds and hangs the whole server
+# The file names and locations are VERY important here
+# DO NOT EDIT THIS UNLESS YOU KNOW WHAT YOU ARE DOING
+# -aa
+name: 'Render Nanomaps'
+on:
+  push:
+    branches: master
+    paths:
+      - 'maps/**'
```

Comment:
I think this will work, but uh... wait till the next map change and worry if it doesn't? x3